src/main/java/com/google/sps/servlets/FetchForumServlet.java

```diff
@@ -49,7 +49,7 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     String query = "SELECT * FROM Question "
         + "LEFT JOIN (SELECT question_id, COUNT(follower_id) followers FROM QuestionFollower "
         + "GROUP BY question_id) CountTable ON Question.id=CountTable.question_id "
-        + "LEFT JOIN (SELECT name, id AS asker_id FROM User) NameTable "
+        + "LEFT JOIN (SELECT username, id AS asker_id FROM User) NameTable "
         + "ON Question.asker_id=NameTable.asker_id "
         + "LEFT JOIN (SELECT question_id, COUNT(id) answers FROM Answer "
         + "GROUP BY question_id) AnswerTable ON Question.id=AnswerTable.question_id;";
```

Comment:
Below, you should make the integers you're referencing in 'queryResult.getString(X)' constants in a Constants.java file or something similar. They could be something like QUESTION_TITLE_INDEX or QUESTION_TITLE_COLUMN so it's more readable, but mostly so it's harder to make a mistake.

Reply:
done!